src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewEntityZeroOrMany.razor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+using System.Text;
+using System.Linq;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class ViewEntityZeroOrMany : ComponentBase
```

Comment:
Same here, let's discuss if we need it.

Reply:
ok I agree I have removed it - and we can just use SelectEntityModel.Name for display